blocks/extensions/plot.py

```diff
@@ -58,6 +58,10 @@ class Plot(SimpleExtension):
         it down you will lose your plots. If you want to store your plots,
         start the server manually using the ``bokeh-server`` command. Also
         see the warning above.
+    server_url : str, optional
+        Url that the bokeh-server is listening to. Ex: when starting the
+        bokeh-server with ``bokeh-server --ip your.i.p.here``, server_url
+        should be ``http://your.i.p.here:5006``.
```

Comment:
Are you sure this is correct? I thought that `--ip` specified the IP being listened on. So that means that if I start `bokeh-server --ip 0.0.0.0` on `alice`, I will have to pass `server_url=alice` on `bob` (and not `0.0.0.0`, which simply means that `alice` will respond to requests originating from any IP address, unlike the default `127.0.0.1`, which listens only to `localhost`).